angular-demo/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/angular-demo",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

No need to rev the version of these examples.

```suggestion
  "version": "0.1.0",
```